dags/benchmark/pytorchxla_torchbench.py

```diff
@@ -19,31 +19,111 @@
 from configs import composer_env, vm_resource
 from configs.benchmark.pytorch import pytorchxla_torchbench_config as config
 
-_MODELS = [
-    "BERT_pytorch",
-    "Background_Matting",
-    "hf_Bert",
-    "Super_SloMo",
-    "tts_angular",
-]
+# Schudule the job to run once per two days at 5:00PM UTC.
+SCHEDULED_TIME = "0 17 */2 * *" if composer_env.is_prod_env() else None
+NETWORK_PREFIX = "projects/tpu-prod-env-automated"
+V5_NETWORKS = f"{NETWORK_PREFIX}/global/networks/mas-test"
+V5E_SUBNETWORKS = f"{NETWORK_PREFIX}/regions/us-east1/subnetworks/mas-test"
+V5P_SUBNETWORKS = f"{NETWORK_PREFIX}/regions/us-east5/subnetworks/mas-test"
 
-# Schudule the job to run every day at 5:00PM UTC.
-SCHEDULED_TIME = "0 17 * * *" if composer_env.is_prod_env() else None
 
 with models.DAG(
     dag_id="pytorchxla-torchbench",
     schedule=SCHEDULED_TIME,
     tags=["pytorchxla", "nightly", "torchbench"],
-    start_date=datetime.datetime(2023, 8, 29),
+    start_date=datetime.datetime(2024, 1, 1),
     catchup=False,
 ) as dag:
-  for model in _MODELS:
-    torchbench_extra_flags = [f"--filter={model}"]
-    config.get_torchbench_config(
-        tpu_version=4,
-        tpu_cores=8,
-        tpu_zone=vm_resource.Zone.US_CENTRAL2_B.value,
-        model_name=model,
-        time_out_in_min=60,
-        extraFlags=" ".join(torchbench_extra_flags),
-    ).run()
+  model = "BERT_pytorch"
+  model = "all"
+  torchbench_extra_flags = [f"--filter={model}"]
+  # Running on V4-8:
+  config.get_torchbench_tpu_config(
+      tpu_version="4",
```

Comment:
Nit: can we reuse version here to avoid potential typo? https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/configs/vm_resource.py#L43

Reply:
Updated along with https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/45#discussion_r1445409968.